CMakeLists.txt

```diff
@@ -109,6 +114,11 @@ elseif("${BACKEND}" STREQUAL "OCL" OR "${BACKEND}" STREQUAL "OPENCL" OR "${BACKE
     find_package(OpenCL QUIET)    # Find package OpenCL
     if(OpenCL_FOUND)
 
+        # Set external flag to notify GPU_SUPPORT and OPENCL backend availability - Compatible with rocAL and MIVisionX
+        add_definitions(-DGPU_SUPPORT)
+        set(RPP_BACKEND_OPENCL 1) # For cmakedefine01 in rpp_backend.h.in
```

Comment:
don't you need to set RPP_BACKEND_HIP 0 here

Reply:
Similarly, the include/rpp_backend.h looks like this for the OpenCL install:
```
#ifndef GUARD_RPP_BACKEND_H_IN
#define GUARD_RPP_BACKEND_H_IN

#define RPP_BACKEND_HIP 0
#define RPP_BACKEND_OPENCL 1

#endif
```